### ISSUE TITLE
Having problem undestanting UseOnlyWhitelistedNamespaces sniff

### ISSUE THREAD

**Issue by lucraraujo** [slevomat/coding-standard#1521]
I have a Laravel application and the 'SlevomatCodingStandard.Namespaces.UseOnlyWhitelistedNamespaces.NonFullyQualified' is triggered for every 'use' statement.

For example:
```php
<?php

declare(strict_types=1);

namespace App\Http\Controllers;

use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
use Illuminate\Foundation\Validation\ValidatesRequests;
use Illuminate\Routing\Controller as BaseController;

//phpcs:disable SlevomatCodingStandard.Classes.TraitUseSpacing.IncorrectLinesCountBeforeFirstUse
//phpcs:disable SlevomatCodingStandard.Classes.TraitUseSpacing.IncorrectLinesCountAfterLastUse
final class Controller extends BaseController
{
    use AuthorizesRequests;
    use ValidatesRequests;
}
```
I get the error in all 3 use statements at the beginning of the file.

I think that I don't really understand what this rule is meant for.

**Comment by kukulich** [comment]
What's your configuration for the sniff?
